rules/integrations/o365/impact_microsoft_365_potential_ransomware_activity.toml

```
@@ -16,8 +16,8 @@ false_positives = [
     represent an adverse encryption process.
     """,
 ]
-from = "now-30m"
-index = ["filebeat-*", "logs-o365*"]
+from = "now-9m"
+index = ["logs-o365.audit-*", "filebeat-*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "Microsoft 365 Potential ransomware activity"
```

```suggestion
name = "Microsoft 365 Potential ransomware activity"
```
The lowercase "ransomeware activity" in a rule title really should be capitalized